src/commands/release/import.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2016-2024 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Flags, Args } from '@oclif/core';
+import Command from '../../command';
+import * as cf from '../../utils/common-flags';
+import { getBalenaSdk, stripIndent } from '../../utils/lazy';
+import { apply } from '@balena/release-bundle';
+import type { ReadStream } from 'fs';
+import { promises as fs } from 'fs';
+import { ExpectedError } from '../../errors';
+
+export default class ReleaseImportCmd extends Command {
+	public static description = stripIndent`
+		Imports a release from a file to an app or fleet.
+
+		Imports a release from a file to an app or fleet. The revision field of the release
+		is automatically omitted when importing a release. The backend will auto-increment
+		the revision field of the imported release if a release exists with the same semver.
+		A release will not be imported if a successful release with the same commit already
+		exists.
+
+		To export a release to a file, use 'balena release export'.
```

Comment:
Perhaps we should add this same line (but for `import` of course) to the `export` command description?

Reply:
Good suggestion.